ganga/GangaLHCb/Lib/Tasks/LHCbTransform.py

```diff
@@ -197,11 +199,30 @@ def createChainUnit(self, parent_units, use_copy_output=True):
                             for pat in excl_pat_list:
                                 if re.search(pat, f.lfn) and "LFN:" + f.lfn in flist:
                                     flist.remove("LFN:" + f.lfn)
+                    elif isType(f, LocalFile):
```

Comment:
Should istype be isinstance instead to allow for inheritance?

Reply:
yes

Reply:
Actually no - these files are wrapped up in proxies so `isinstance` doesn't work on them